packages/backend/src/peripherals/database/transactions/UserTransaction.ts

```diff
@@ -21,8 +21,9 @@ export type UserTransactionData =
   | ForcedTradeData
   | ForcedWithdrawalData
   | FullWithdrawalData
-  | EscapeVerifiedData
   | WithdrawalPerformedData
+  | EscapeVerifiedData
```

Comment:
It turns out that parameters to `verifyEscape()` and `escape()` are different for spot and perpetual, even though the whole process is the same.

Perpetual:
```    
function verifyEscape(
        uint256[] calldata merkleProof,
        uint256 nAssets,
        uint256[] calldata sharedState
    )
    
 function escape(
        uint256 starkKey,
        uint256 vaultId,   // <-- this acutally is a positionId
        uint256 quantizedAmount
    )
 ```

Spot:
    
```
function verifyEscape(
          uint256[] calldata escapeProof
)     

function escape(
        uint256 ownerKey,
        uint256 vaultId,
        uint256 assetId,
        uint256 quantizedAmount
    )
```

Because of that, and to make things consistent, I suggest renaming the UserTransaction types and SentTransaction types to:

* VerifyPerpetualEscape
* FinalizePerpetualEscape

* VerifySpotEscape
* FinalizeSpotEscape

and to modify all function names accordingly (e.g. encode... etc.)

Reply:
Well, I agree we need that refactoring, although I would do this as a part of spot escapes PR